packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -226,13 +226,21 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
                 async () => summarizer.run(clientId, this.summarizerOptions),
             );
         }).catch((error) => {
-            this.logger.sendErrorEvent({ eventName: "SummarizerException" }, error);
-            this.emit("summarizerWarning", error);
-
-            // Note that summarizer may keep going (like doing last summary).
-            // Ideally we await stopping process, but this code path is due to a bug
-            // that needs to be fixed either way.
-            this.stop("summarizerException");
+            // Most of exceptions happen due to container being closed while loading it.
+            // Not worth reporting such errors. That said, we might miss some real errors if
+            // we ignore blindly, so try to narrow signature we are looking for - skip logging
+            // error only if interactive container also is disconnected (or went through reconnection),
+            // we do not have summarizer. We could add error.fluidErrorCode !== "containerClosedWithoutErrorDuringLoad",
+            // but that does not seem to be necessary.
+            if (this.getShouldSummarizeState().shouldSummarize || this.summarizer !== undefined) {
```

Comment:
I think the comment above is a bit confusing coupled with `this.getShouldSummarizeState().shouldSummarize`. `shouldSummarize` can be `true` in this context only if the container is connected, right?

Reply:
Correct. Which part is confusing? Is this more readable:

const skip = !this.getShouldSummarizeState().shouldSummarize && this.summarizer == undefined;
if (!skip) {

Reply:
No, the code is clear, the comment is confusing because it mentions `container also is disconnected`

Reply:
how about now? Is it better?